EnglishFilter/sections/specific.txt

```diff
@@ -595,6 +595,7 @@ swordz.io##div[style="width:300; height: 250; background-color: #000"]
 disasterscans.com##div[id^="media_image-"].no-icon.heading-style-1
 9tsu.net##.ads-system
 pcworld.com###amazon-bottom-widget
+shiachat.com##tr > td > div[class^="ipsA"]
```

Comment:
Try to use more easy way to block it than you have. Something like this `shiachat.com##body > div#ipsLayout_header + center` and additionally block gif images via URL

Reply:
I've modified the rule, but i am not sure about gifs. Many of them are userpics.

Reply:
I don't mean all the gifs, just those ads.
For example, you can block these urls 
```
https://www.shiachat.com/forum/uploads/monthly_2020_07/1664101338_alaligemsa.gif.778f6887c1477add141fc4129dfded51.gif
https://www.shiachat.com/forum/uploads/monthly_2015_11/shiamatch.gif.a9bfbde53248764fbf95258566652208.gif
```